Bugzilla/Teams.pm

```diff
@@ -24,6 +26,7 @@ use Try::Tiny qw(try catch);
 use Type::Utils;
 use Types::Standard qw(:types);
 
+# DEPRECATED: Will be removed in future release
```

Comment:
As far as I can tell these methods won't be used with this change; should be safe to delete them now rather than at a later date.